test/decl/protocol/special/coding/Inputs/class_codable_simple_multi1.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: objc_interop
```

Comment:
So to clarify -- this code does work on Linux with corelibs-foundation right? It's just that it's hard to test in this scenario?

Would adding some tests that rely on the conformance being derived to swift-corelibs-foundation be an acceptable solution then?